packages/syntax-core/src/Typography/Typography.tsx

```diff
@@ -127,6 +127,20 @@ const Typography = forwardRef<
      * @defaultValue "regular"
      */
     weight?: "regular" | "medium" | "semiBold" | "bold";
+    /**
+     * Controls how whitespace within the element is handled.
+     *
+     * See: https://developer.mozilla.org/en-US/docs/Web/CSS/white-space
+     *
+     * @defaultValue "inherit"
+     */
+    whiteSpace?:
```

Comment:
hmm. generally, we've limited alot of these props to only the ones we need. ive inherited christian's rather be more strict and then if we need the others ones we'll add it later

how often do you think we'll need all of these? 
i imagine just inherit, normal, nowrap and pre will be fine

Reply:
For the current use case we want `pre-line` so that we can preserve line breaks but still wrap.
![Screenshot 2024-10-22 at 5 15 10 PM](https://github.com/user-attachments/assets/9fdb6f49-f33c-40b3-b393-1ef0fb4da35a)

I'll limit it to `inherit`, `normal`, `nowrap` and `pre-line` for now.

Unless you think there's a use case for `pre` as well:

![Screenshot 2024-10-22 at 5 07 51 PM](https://github.com/user-attachments/assets/ee7e39dc-251e-4686-8fd1-55e530c3f489)

Reply:
just the ones you mentioned are fine. thanks!

Reply:
> just the ones you mentioned are fine. thanks!

Done!